tlj-ui-web/src/main/webapp/WEB-INF/view/login_page.jsp

```diff
@@ -14,6 +15,9 @@ pageEncoding="utf-8"%>
 <form action="processingForm" method="POST">
 <p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Login:<input type="text" name="login" maxlength="50" size="30"></p>
 <p>Password:<input type="password" name="password" maxlength="50" size="30"></p>
+<%if(request.getAttribute("wrong_user") != null) {%>
```

Comment:
Why do you not use **jstl** library(how it was here: https://github.com/DzmitryBasiachenka/tiny-life-journal/blob/bbd8dd79b5af65319675f556ae513ede606cf3bc/tlj-ui-web/src/main/webapp/WEB-INF/tiles/pictures-list-jstl.jsp)?